src/axom/quest/IntersectionShaper.hpp

```diff
@@ -33,8 +33,8 @@
 
 #include "mfem.hpp"
 
-#include "fmt/fmt.hpp"
-#include "fmt/locale.h"
+#include "axom/fmt.hpp"
```

Comment:
=/ my bad

Reply:
I think it was merged after yours and there was an invalid guard in CMake preventing these files from being included